sys/include/xtimer.h

```diff
@@ -8,18 +8,31 @@
  */
 
 /**
- * @defgroup  sys_xtimer Timers
- * @ingroup   sys
- * @brief     Provides a high level timer module to register
- *            timers, get current system time, and let a thread sleep for
- *            a certain amount of time.
+ * @defgroup    sys_xtimer xtimer high level timer abstraction layer (deprecated)
+ * @ingroup     sys
+ * @brief       Provides a high level timer module to register
+ *              timers, get current system time, and let a thread sleep for
+ *              a certain amount of time.
  *
  * The implementation takes one low-level timer and multiplexes it.
  *
  * Insertion and removal of timers has O(n) complexity with (n) being the
  * number of active timers.  The reason for this is that multiplexing is
  * realized by next-first singly linked lists.
  *
+ * @warning     xtimer has been deprecated in favor of the @ref sys_ztimer
```

Comment:
Why not use the `@deprecated` command?

Reply:
This way it will show up in https://doc.riot-os.org/deprecated.html

Reply:
Fixed